src/schemas/json/project.json

```diff
@@ -1,11 +1,68 @@
 {
-	"title": "JSON schema for DNX project.json files",
-	"$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
The $schema directive should be left in - the schema spec indicates that documents without $schema directives (pointing at a meta-validation schema) should be assumed to point at "latest" which could eventually be a problem